inlang.config.js

```diff
@@ -0,0 +1,19 @@
+export async function defineConfig(env) {
+	const { default: i18nextPlugin } = await env.$import(
+		"https://cdn.jsdelivr.net/npm/@inlang/plugin-i18next@2/dist/index.js",
+	)
+  const { default: standardLintRules } = await env.$import(
+		"https://cdn.jsdelivr.net/npm/@inlang/plugin-standard-lint-rules@3/dist/index.js",
+	)
+
+	return {
+		referenceLanguage: "en-US",
+		plugins: [
+			i18nextPlugin({
+				pathPattern: "./packages/mask/shared-ui/locales/{language}.json",
```

Comment:
we have many locales folder, if you configured it like this, it won't work for other monorepo projects